src/components/FormGroup/FormGroup.test.ts

```diff
@@ -10,6 +10,12 @@ describe('template', () => {
 
     componentSlotRenderTest(FormGroup);
 
+    componentSlotRenderTest(FormGroup, 'labelAppend', {
```

Comment:
No update in HTML snapshot?

Reply:
Nope, because the test doesnt have a label prop, the `<label>` with the slot in it doesnt get rendered. So the slot also doesnt get rendered

Reply:
Huh.. then whats the point of the test if not to check that the slot is rendered?

Reply:
`componentSlotRenderTest` is not making a snapshot.